lib/src/main/java/org/team100/lib/indicator/LEDStrip.java

```diff
@@ -5,6 +5,11 @@ public class LEDStrip {
     private final int m_length;
     private final int m_offset;
 
+    public enum strip{
```

Comment:
it is very confusing for the enum that represents "which side of the robot is the light facing" to be called "strip".  you should always try to name things according to what they actually mean.